packages/core/workflows-sdk/src/helper/workflow-export.ts

```diff
@@ -9,9 +9,9 @@ import {
 import { Context, LoadedModule, MedusaContainer } from "@medusajs/types"
 import {
   ContainerRegistrationKeys,
+  isPresent,
```

Comment:
what kind of sorting does your IDE do? 🤣

Reply:
I believe it is alphabetically sorting 😂

Reply:
case-insensitive

Reply:
Yes just alphabetical 😅 i can have a look later if i can refine